src/modules/ChannelList/components/ChannelPreview/utils.js

```diff
@@ -53,16 +54,23 @@ export const getTotalMembers = (channel) => (
 );
 
 const getChannelPreviewFileDisplayString = (mimeType) => {
+  const stringSet = LabelStringSet;
```

Comment:
We should use the `useLocalization` instead of LabelStringSet.

Reply:
e.g. https://github.com/sendbird/sendbird-uikit-react/blob/main/src/modules/ChannelList/components/ChannelPreview/index.tsx#L75

Reply:
thank you!

Reply:
ah no we cannot do this. dispatcher is undefined in tests.

Reply:
<img width="1330" alt="Screen Shot 2023-10-17 at 1 52 00 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/16806397/dffc2cdd-6eeb-41bd-8089-b704959fec2f">

Reply:
You can pass stringSet to `getChannelPreview~` function and use them in the components.

Reply:
Not sure what you mean. Could you help to push a commit?